package.json

```diff
@@ -1,11 +1,9 @@
 {
   "name": "lekplats",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
The bump in the version should not be part of the PR, but instead it should be bumped after it is merged.

Reply:
Ok. Will remove it.